$OpenBSD: patch-gexiv2_meson_build,v 1.2 2018/12/06 22:47:28 naddy Exp $

https://gitlab.gnome.org/GNOME/gexiv2/issues/27

Replace empty string with empty array in order to pass no argument
rather than an empty one; clang ignores empty arguments, but gcc
errors out.

Index: gexiv2/meson.build
--- gexiv2/meson.build.orig
+++ gexiv2/meson.build
@@ -35,8 +35,8 @@ install_headers(gexiv2_headers, subdir : 'gexiv2')
 
 # Static map file
 mapfile = 'gexiv2.map'
-if host_machine.system() == 'darwin'
-    vflag = ''
+if ['darwin', 'openbsd'].contains(host_machine.system())
+    vflag = []
 else
     vflag = '-Wl,--version-script,@0@/@1@'.format(meson.current_source_dir(), mapfile)
 endif
